### PR TITLE
extras/po: Some updates to the german translation

### PR DESCRIPTION
**Pull request by Narrat**

### CODE REVIEW

**Comment by jaromil** [comment]
and again thanks! just waiting for @gdrooid to have a look, you there?

File: extras/po/de.po [hunk 1]
```diff
@@ -174,31 +173,31 @@ msgstr " // Gräber bearbeiten:"
...
 
 #: tomb:Commandline interaction:usage:604
 msgid " index   update the search indexes of tombs"
-msgstr " index   Den Suchindex der Gräber aktualisieren"
+msgstr " index   die Suchindizes der Gräber aktualisieren"
 
 #: tomb:Commandline interaction:usage:605
 msgid " search  looks for filenames matching text patterns"
-msgstr " search  Sucht nach Dateinamen welche Textmustern entsprechen"
+msgstr " search  sucht nach Dateinamen welche Textmustern entsprechen"
```
Comment (gdrooid) [inline extras/po/de.po]: Why make this one lowercase? Others are left with an uppercase first letter.

**Comment by gdrooid** [comment]
It looks good to me except for a minor inconsistency in the commandline usage message, where the description of some explanations starts with a lowercase letter and in others with an uppercase letter. Is that just inconsistent, or does it just look weird to me but is how it's supposed to be in German?

PS: @jaromil I'm here, yeah, although I've been pretty busy/tired since I got a job. :/ I hope I can be more responsive from now on.

**Comment by Narrat** [comment]
oddity with the German language. Upper case words are normally noun words.
So the sentences start most of the time with with a verb (lowercase). Exception is the one on line 188 which is the translation of (the) list. or could change the sentence to use the verb of that word

**Comment by gdrooid** [comment]
Alright! I'm merging it then, thank you Narrat.

**Pull request merged**